src/App.js

```diff
@@ -73,29 +46,80 @@ class App extends React.Component {
       this.setState({ newTodo: "" });
     }
   };
+
+  markAsCompleted = (id) => {
+    this.setState(prevState => 
+      ({
+
+      todoList: prevState.todoList.map(task => {
+        if (task.id ===id) {
+          task.completed = !task.completed;
+          task.style ={
+            textDecoration: task.completed ? "line-through" : "none"
+          }
+        }
+        return task;
+      })
+    })
+    )
+  }
+
+  /* deleteDoneTodos() {
+    this.state.todoList = this.state.todoList.filter((element) => element.)
+  } */
+
+  render() {
+    return (
+      <div className="todoListComplete">
+        {this.state.todoList.map(todo => (
+          <div><Paragraph text={todo.task} todo={todo} id2={todo.id} method={this.markAsCompleted}/></div>
+        ))}
+        <form>
+        <input
+          value={this.state.newTodo}
+          onChange={this.changeHandler}
+          /* onKeyDown={this.addTodoEnter} */
+          type="text"
+        />
+        <button onClick={this.addTodo}>Add Task!</button>
+        <button onClick={this.deleteDoneTodos}>Delete done Tasks!</button>
+        </form>
+      </div>
+    );
+  }
 }
 
+
+
+
+
+
 class Paragraph extends React.Component{
 
   constructor(props) {
     super(props);
     this.state = {
       isDone: false,
-      content: this.props.content
+      content: props.text,
     };
   }
 
-  style() {
-    if(this.state.isDone){
-      return {textDecoration: "none"}
-    }
-    else{
-      return {textDecoration: "line-through"}
+  style= () => {
+    return {
+      textDecoration : this.state.isDone ? "line-through" : "none" ,
     }
   }
 
+  crossOut = () => {
+    this.props.method(this.props.todo.id);
+  }
+
+  toDelete= () => {
+    return this.state.isDone;
+  }
+
   render(){
-    return <p>{this.content}</p>
+    return <p onClick={this.crossOut}>{this.state.content}</p>
   }
 
```

Comment:
- Inaki, great effort completing the MVP for the project ! 
- Your code is clean and You are properly splitting up the code into multiple components